apps/ledger-live-desktop/src/renderer/components/LiveAppDrawer.tsx

```diff
@@ -60,6 +60,17 @@ export function isStartExchangeData(data: unknown): data is StartExchangeData {
   return "exchangeType" in data;
 }
 
+class DrawerClosedError extends Error {
```

Comment:
Is it not a bit of overhead to create a class to generate an Error?

Reply:
We should probably just use our `createCustomErrorClass` for this also